examples/plot_OT_1D_smooth.py

```diff
@@ -110,3 +110,15 @@
 ot.plot.plot1D_mat(a, b, Gsm, 'OT matrix Smooth OT l2 reg.')
 
 pl.show()
+
+#%% Sparsity-constrained OT
```

Comment:
you cold rename the exmaple "Smooth and sparse OT" and maybe remove sinkhorn and KL to fous on the new solvers more?